www/js/menu.js

```diff
@@ -29,6 +45,45 @@ const goToShabadPage = (shabadId) => {
   document.querySelector('#shabad-pageLink').click();
 };
 
+// format the date default to "Month Day, Year"
+const formatDate = (dateString, format = 'LL') => moment(dateString).format(format);
+
+const stripScripts = (string) => {
+  const div = document.createElement('div');
+  div.innerHTML = string;
+  const scripts = div.getElementsByTagName('script');
+  let i = scripts.length;
+  while (i > 0) {
+    scripts[i].parentNode.removeChild(scripts[i]);
+    i -= 1;
+  }
+  return div.innerHTML;
+};
+
+const scriptTagCheckRegEx = /<[^>]*script/i;
+
+const parseContent = (contentString) => {
+  if (scriptTagCheckRegEx.test(contentString)) {
```

Comment:
Stripping script tags might be over kill....if we trust the source then we can leave this check/filter out